### ISSUE TITLE
Customizing animals and colors 

### ISSUE THREAD

**Issue by iccir** [andreaslarsen/animalnoteheads#12]
One idea that I was thinking about from our "the colors are too close" reddit discussion: there are students with deficient color vision (color blindness).  There are also students where certain color choices and shape combinations may not be obvious due to cultural difference (for example, an English-speaking region where a big red dog isn't obvious may want to use alternate dog icon / color).

You are already seeing this to some extend with the language support.

One idea would be to allow teachers to create their own set of seven animals, with their own color customizations.

Note: This is probably an idea for the far future and not v1.0. :)

**Comment by larsenwork** [comment]
Yep, my long term ambition is that all the sheets on the website can be color adjusted as you like - probably leveraging a browser plugin that allows for private custom css. And then have a css color template bank optimised for various kinds of dyschromatopsia. 

But I'm trying to do MVP and fokus on the 90% first so this get's added to the wishlist milestone :smile:
